synapse/tests/test_lib_config.py

```diff
@@ -159,6 +164,24 @@ def __init__(self, proxy):
                     self.eq(tufo[1]['tufo:form'], 'inet:ipv4')
                     self.eq(tufo[1]['inet:ipv4'], 0)
 
+    def test_configable_noncallable(self):
+
+        with self.getRamCore() as core:
+            with s_daemon.Daemon() as dmon:
+                dmon.share('core', core)
+                link = dmon.listen('tcp://127.0.0.1:0/core')
+                port = link[1]['port']
+                core_url = 'tcp://127.0.0.1:%d/core' % port
+                conf = {
+                    'ctors': (
+                        ('core', 'ctor://synapse.cortex.openurl("%s")' % core_url, {}),
+                        ('cc', 'ctor://synapse.tests.test_lib_config.MyCoolClass(core)', {}),
+                    ),
+
+                }
+                with s_daemon.Daemon() as inner_daemon:
+                    inner_daemon.loadDmonConf(conf)
```

Comment:
This raises `TypeError: 'Proxy' object is not callable` before the fix as the code calls `pass` instead of `continue` as the method will still be added even though it is not callable.